ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -853,22 +858,65 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
         DisplacementDim>(_element, _is_axially_symmetric, p,
                          *_process_data.pressure_interpolated);
 
+    int const elem_id = _element.getID();
+    ParameterLib::SpatialPosition x_position;
```

Comment:
The lines 862--865 could be move into the if condition block starting in 869.

Reply:
We agreed to go without the optional output, so the if condition isn't there and this doesn't apply anymore. But before you are totally correct.

Reply:
Ok.